sql/alps_submitter.go

```diff
@@ -376,10 +402,21 @@ func generateEstimatorCreator(estimator string, attrs []*attribute) (string, err
 		}
 		cl[idx] = code
 	}
+	if args != nil {
+		for _, arg := range args {
+			cl = append(cl, arg)
+		}
+	}
 	return fmt.Sprintf("tf.estimator.%s(%s)", estimator, strings.Join(cl, ",")), nil
 }
 
 func newALPSTrainFiller(pr *extendedSelect) (*alpsFiller, error) {
+	scratchDir, err := ioutil.TempDir("/tmp", "alps_scratch_dir_")
```

Comment:
Should we remove `scratchDir` after the training?

Reply:
The `scratchDir` will be deleted until the output model is saved using `sqlfs` which is blocking by `ODPS DB`.

Reply:
Would you please at least add a TODO?

Reply:
Ok, thanks